examples/tv-app/tv-common/clusters/channel/ChannelManager.h

```diff
@@ -72,5 +73,6 @@ class ChannelManager : public ChannelDelegate
 
 private:
     // TODO: set this based upon meta data from app
-    uint32_t mDynamicEndpointFeatureMap = 3;
+    static constexpr uint32_t kEndpointFeatureMap = 3;
```

Comment:
Need feature map explanations. This seems to apply to the entire PR, will stop calling it out however please update all endpoint feature maps.

Reply:
Created an issue
https://github.com/project-chip/connectedhomeip/issues/31991